editor/components/block-list/block.js

```diff
@@ -434,11 +426,7 @@ export class BlockListBlock extends Component {
 							<BlockHtml uid={ block.uid } />
 						) }
 						{ ! isValid && [
-							createElement( blockType.save, {
-								key: 'invalid-preview',
-								attributes: block.attributes,
-								className,
-							} ),
+							getSaveElement( blockType, block.attributes ),
```

Comment:
Won't we have a React warning here about the missing key?

Reply:
I didn't notice any. See example:

![screen shot 2017-12-18 at 11 40 15](https://user-images.githubusercontent.com/699132/34102139-4a0f6644-e3e8-11e7-9856-2ea1190530d3.png)

I will add one to be on the safe side.

Reply:
Addressed with https://github.com/WordPress/gutenberg/pull/4009/commits/3650811f405ae91d1563d83195b307e37067743d.